src/main/webapp/app/overview/course-conversations/course-wide-search/course-wide-search.component.ts

```diff
@@ -74,6 +76,7 @@ export class CourseWideSearchComponent implements OnInit, AfterViewInit, OnDestr
         public metisConversationService: MetisConversationService, // instance from course-conversations.component
         private formBuilder: FormBuilder,
         public cdr: ChangeDetectorRef,
+        private courseSidebarService: CourseSidebarService,
```

Comment:
```suggestion
        private courseSidebarService = inject(CourseSidebarService)
```

Reply:
Thanks for the feedback! I addressed it in this [commit](https://github.com/ls1intum/Artemis/pull/9460/commits/068c060c09808233eb33695f12486d82466e22dd).